RTN-008.tex

```diff
@@ -62,114 +63,113 @@
+In this scenario, the Rubin Operations team would undertake a best-effort activity to produce direct image sources, \DIASources,  and \DIAObjects (if possible) for the TOO observations.
 The goal would not be to perform scientific analysis\footnote{though team members would not be precluded from doing so later} but to rapidly report reliable results to the community.
 The team would use its judgment to undertake custom processing to enable the rapid identification of plausible counterparts.
 This processing might include image-to-image differencing using prior LSST data, on-the-fly generation of new templates, or various flavors of forced photometry.
 
 While it might be possible to generate functional alert packets to forward to community alert brokers, the team would also likely report results via other channels, in particular the GCN Circulars used by the broader EM/GW community.
 As an extremely time-sensitive, alert-like product these results would be made world-public immediately.
 
-LSST's TOOs will be highly visible, highly scrutinized observations, and so it will be valuable to have data processing performed by the highly-experience LSST operations team under the LSST imprimatur.
+Rubin's TOOs will be highly visible, highly scrutinized observations, and so it will be valuable to have data processing performed by the experienced Rubin operations team under the Rubin imprimatur.
 Successful execution of such processing will require a specialized team that will agree to be on-call in case of TOO triggers.
-This team would consist primarily of LSST Operations personnel but could include key members of the Science Collaborations with relevant expertise (emphasizing again that the purpose of this activity is data processing, not science analysis).
+This team would consist primarily of Rubin personnel but could include members of the commissioning team with relevant expertise (emphasizing again that the purpose of this activity is data processing, not science analysis).
 
 It will be crucial to develop and rehearse procedures in advance to clarify authority, responsibility, and communications channels. 
 In developing these procedures it would be useful to consult with the operations of other public facilities conducting EM/GW followup, such as \textit{Swift} and \textit{Fermi}.
 
 During O4 the on-call team should be activated even in cases when templates exist for the TOO observations.
 In this case custom processing might not be required, but the team would perform detailed QA of the alerts produced by the standard pipeline and report any problems with the automated analysis to the community via GCN.
 
+We assume that TOO observations are relatively uncommon in this period (restricted to no more than 1--2 triggers/month, on average, perhaps) such that the impacts on on-call staff are tolerable.  
```

Comment:
Add: Criteria to trigger during high priority commissioning activities will be worked out with the commissioning team and project management.